ui/everystudent/src/main/java/org/cru/godtools/everystudent/EveryStudentView.java

```diff
@@ -51,7 +51,10 @@ public void onCreate(Bundle savedInstanceState) {
         SharedPreferences settings = getSharedPreferences(Constants.PREFS_SETTINGS, 0);
         if (settings.getBoolean("wakelock", true)) {
             PowerManager pm = (PowerManager) getSystemService(Context.POWER_SERVICE);
-            wl = pm.newWakeLock(PowerManager.FULL_WAKE_LOCK, "DoNotDimScreen");
+            // Lint Check requires a prefix. I will use using gdt: - Gyasi Story
```

Comment:
Is there some significance to "gdt"? Just my own curiosity.

Reply:
No I was thing of a shorten prefix for GodTools.  It can be changed if you link.

Reply:
Nope, I think it's fine. I just wasn't sure if it was an abbreviation for something in the Android SDK or something else.